src/Location/Location.tsx

```diff
@@ -12,107 +12,125 @@ import wind from '../assets/wind.png'
 // feel free to break up lines of text for readability, i'm getting tired
 
 import { getAllData } from '../helpers/dataCleaner'
+import { getTestData } from '../helpers/apiCalls'
+
+// interface LocationProps {
+// 		date?: string,
+// 		day?: string,
+// 		time?: number,
+// 		city?: string;
+// 		state?: string,
+// 		country?: string,
+// 		temp?: number,
+// 		aqi?: number,
+// 		uvIndex?: number,
+// 		visibility?: number,
+// 		wind?: number,
+// 		windDirection?: string,
+// 		precipitation?: number,
+// 		allergyLevel?: number,
+// 		discussion?: string,
+//   }
 
 interface LocationProps {
-		date?: string,
-		day?: string,
-		time?: number,
-		city?: string;
-		state?: string,
-		country?: string,
-		temp?: number,
-		aqi?: number,
-		uvIndex?: number,
-		visibility?: number,
-		wind?: number,
-		windDirection?: string,
-		precipitation?: number,
-		allergyLevel?: number,
-		discussion?: string,
-  }
+	// getMatchedData: Function
+	query: string
+	matchDetails: Array<any>
+}
 
 // const Location = ({locationData, currentWeather, weeklyWeather, currentAir}) => {
-// 	const [allData, setAllData] = useState()
+const Location: React.FC<LocationProps> = props => {
+	const [allData, setAllData] = useState({})
+
+	// setAllData(getAllData('denver'))
+
+	const getMatchedData = () => {
+		console.log('hi i\'m a match')
+		console.log('put me into getAllData please', props.matchDetails)
+	}
+
+	useEffect(() => {
+		getMatchedData()
+		// setAllData(getAllData(props.matchDetails[0], props.matchDetails[1]))
```

Comment:
This will call getAllData upon page load with the proper coordinates based on the result card that the user clicks. It will set allData with ALL of the information though as well. You can then use allData to pull out anything you want from it and pass that into the weeklyData or just anything within the Location render.